pkg/plugins/register.go

```diff
@@ -0,0 +1,19 @@
+package plugins
+
+import (
+	"sigs.k8s.io/gateway-api-inference-extension/pkg/epp/plugins"
+
+	"github.com/llm-d/llm-d-inference-scheduler/pkg/plugins/filter"
+	"github.com/llm-d/llm-d-inference-scheduler/pkg/plugins/scorer"
+)
+
+// RegisterAllPlugins registers the factory functions of all plugins in this repository.
+func RegisterAllPlugins() {
+	plugins.Register(filter.ByLabelsFilterType, filter.ByLabelFactory)
```

Comment:
why do we register an example filter only for documentation purpose?

Reply:
I registered all of the plugins